lib/pairmotron/pair_builder.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Determination do
+  defstruct dead_pairs: [], remaining_pairs: [], available_users: []
```

Comment:
Can we give this a module_doc that spells out exactly what each field is, and what it's intended to be used for?